File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management_specs/MagicCarpetTest.cs
using NUnit.Framework;

namespace WizardManagement { 

  [TestFixture]
  public class MagicCarpetTest {
    MagicCarpet magicCarpet;

    [SetUp]
    public void Init() 
    {
      magicCarpet = new MagicCarpet("Yellow");
    }

    [Test]
    public void HasColour()
    {
      Assert.AreEqual("Yellow", magicCarpet.Colour);
    }

    [Test]
    public void CanFly()
    {
      Assert.AreEqual("Hovering up, straightening out, flying off!", magicCarpet.Fly());
    }
  }
}

File: week_06/day_3/composition/Composition_Start/app/src/test/java/example/codeclan/com/composition_start/MagicCarpetTest.java
package example.codeclan.com.composition_start;

/**
 * Created by user on 28/08/2017.
 */

import static org.junit.Assert.*;
import org.junit.*;

import example.codeclan.com.composition_start.wizard_management.*;


public class MagicCarpetTest {

    MagicCarpet magicCarpet;

    @Before
    public void before(){
        magicCarpet = new MagicCarpet("Yellow");
    }

    @Test
    public void hasColour(){
        assertEquals("Yellow", magicCarpet.getColour());
    }

    @Test
    public void canFly(){
        assertEquals("Hovering up, straightening out, flying off!", magicCarpet.fly());
    }

}
